components/hab/src/cli.rs

```diff
@@ -545,6 +545,27 @@ pub fn get(feature_flags: FeatureFlag) -> App<'static, 'static> {
                     (ex: core/redis, core/busybox-static/1.42.2/21120102031201)")
                 (@arg NO_DEPS: --("no-deps") "Don't uninstall dependencies")
             )
+            // alas no hyphens in subcommand names..
+            // https://github.com/clap-rs/clap/issues/1297
+            (@subcommand bulkupload =>
+                (about: "Bulk Uploads Habitat Artifacts to a Depot from a local directory.")
+                (aliases: &["bul", "bulk"])
+                (@arg BLDR_URL: -u --url +takes_value {valid_url} "Specify an alternate Depot \
+                    endpoint. If not specified, the value will be taken from the HAB_BLDR_URL \
+                    environment variable if defined. (default: https://bldr.habitat.sh)")
+                (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
+                (@arg CHANNEL: --channel -c +takes_value
+                    "Optional additional release channel to upload package to. \
+                     Packages are always uploaded to `unstable`, regardless \
+                     of the value of this option.")
+                (@arg FORCE: --force "Skips checking availability of package and \
```

Comment:
Sorry for the typo in the LucidChart doc, "Skips" should be "Skip" here.

Reply:
Fixed in https://github.com/habitat-sh/habitat/pull/6958/commits/ae2f142bfe52399952e722020275aa16341d6290

Reply:
👍